### DISCUSSION TITLE
A description of the Synology DSM 7.2 documentation and installation issues

### DISCUSSION THREAD

**Discussion by cukewu** [rustdesk/rustdesk#9774] [Q&A]
I am following the documentation to install using docker compose on Synology DSM 7.2. The link is as follows: https://rustdesk.com/docs/en/self-host/rustdesk-server-oss/synology/dsm-7/

There is a place in the documentation that is not detailed enough and may cause misunderstanding.
You could modify the line with hbbs to your NAS’s LAN IP temporarily just like the picture. After you verify your server is working, you should change back.
In fact, you cannot just fill in the IP here. It should be command: hbbs -r rustdesk.example.com:21117, right?

At the same time, I have a question. My NAS is deployed in the intranet, and the domain name is also resolved to the egress IP of the intranet. Then can I use the LAN IP for hbbs? Because port mapping is done on the router, access from the public network will also be mapped to the port of the intranet NAS through the router. But the document description says
If you have set your hbbs command to point to your LAN IP, and verified it is working, it is time to change to the domain, as it will not work when attempting to use it outside of your LAN.
I don't quite understand why this is, does rustdesk require a domain name?

![1](https://github.com/user-attachments/assets/e4970e50-1bd5-473b-a415-27b7e53ba2f4)

**Comment by xlionjuan** [comment]
Ignore that Note temporary sorry. By default, if no any relay server (`-r` arg) specified, the client will use same address with ID server.

> does rustdesk require a domain name?

Very recommend to have, because domain will not change(as long as you paid), but IP is not, using domain to ensure you won't lose control if your server IP is changed.
